internal/assertions/apiversions_response_assertion.go

```diff
@@ -29,51 +20,99 @@ var errorCodes = map[int]string{
 	100: "UNKNOWN_TOPIC_ID",
 }
 
-func (a ApiVersionsResponseAssertion) Evaluate(fields []string, AssertApiVersionsResponseKey bool, logger *logger.Logger) error {
-	if Contains(fields, "ErrorCode") {
+type ApiVersionsResponseAssertion struct {
+	ActualValue   kafkaapi.ApiVersionsResponse
+	ExpectedValue kafkaapi.ApiVersionsResponse
+	logger        *logger.Logger
+	err           error
+
+	// empty slice = assert all fields (default)
+	// non-empty slice = assert with exclusions
+	excludedBodyFields []string
+}
+
+func NewApiVersionsResponseAssertion(actualValue kafkaapi.ApiVersionsResponse, expectedValue kafkaapi.ApiVersionsResponse, logger *logger.Logger) *ApiVersionsResponseAssertion {
+	return &ApiVersionsResponseAssertion{
+		ActualValue:        actualValue,
+		ExpectedValue:      expectedValue,
+		logger:             logger,
+		excludedBodyFields: []string{},
+	}
+}
+
+// assertBody asserts the contents of the response body
+// Fields asserted by default: ErrorCode
+func (a *ApiVersionsResponseAssertion) assertBody() *ApiVersionsResponseAssertion {
```

Comment:
```suggestion
func (a *ApiVersionsResponseAssertion) assertBody() error {
```

(We don't need chaining for these assert methods anymore)